### DISCUSSION TITLE
GitHub private repositories

### DISCUSSION THREAD

**Discussion by caioagiani** [community/community#23553] [New to GitHub]
<p>What is the limit of private repositories for a github account without a “Pro” subscription?</p>

**Comment by jfullstackdev** [comment, marked as answer]
<p>as you can see, Free GitHub contains unlimited repo, and it’s included at Pro, so yes, enjoy that,</p>
<p></p><div class="lightbox-wrapper"><a class="lightbox" href="https://aws1.discourse-cdn.com/github/original/3X/8/3/838c700f5ec0c8e8fb4d6aa934c0f5e33b07eeb2.png" data-download-href="/uploads/short-url/iLJqYtvlu2P9IgghLxZTBlZ8asi.png?dl=1" title="image"><img src="https://user-images.githubusercontent.com/47092464/181077868-e68f55c6-e0a8-4d93-8f22-b1bd90f77336.png" alt="image" data-base62-sha1="iLJqYtvlu2P9IgghLxZTBlZ8asi" width="690" height="284" srcset="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/838c700f5ec0c8e8fb4d6aa934c0f5e33b07eeb2_2_690x284.png, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/838c700f5ec0c8e8fb4d6aa934c0f5e33b07eeb2_2_1035x426.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/3X/8/3/838c700f5ec0c8e8fb4d6aa934c0f5e33b07eeb2_2_1380x568.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/3X/8/3/838c700f5ec0c8e8fb4d6aa934c0f5e33b07eeb2_2_10x10.png"><div class="meta">
<svg class="fa d-icon d-icon-far-image svg-icon" aria-hidden="true"><use href="#far-image"></use></svg><span class="filename">image</span><span class="informations">1389×573 70.2 KB</span><svg class="fa d-icon d-icon-discourse-expand svg-icon" aria-hidden="true"><use href="#discourse-expand"></use></svg>
</div></a></div>
<p>not only that, you now can now add unlimited collaborators in a private repo</p>
<aside class="onebox allowlistedgeneric" data-onebox-src="https://github.blog/changelog/2020-04-14-private-repositories-with-unlimited-collaborators-available-to-all-github-accounts-and-changes-to-github-paid-plans/">
  <header class="source">
      <img src="https://user-images.githubusercontent.com/47092464/181077869-323c5580-8e37-43b2-828a-73fc4db9b4ad.png" class="site-icon" width="32" height="32">

      <a href="https://github.blog/changelog/2020-04-14-private-repositories-with-unlimited-collaborators-available-to-all-github-accounts-and-changes-to-github-paid-plans/" target="_blank" rel="noopener">The GitHub Blog</a>
  </header>

  <article class="onebox-body">
    <div class="aspect-image" style="--aspect-ratio:690/362;"><img src="https://github.blog/wp-content/uploads/2022/01/Product-Cloud-Enterprise@2x.png?fit=2400%2C1260" class="thumbnail" width="690" height="362"></div>

<h3><a href="https://github.blog/changelog/2020-04-14-private-repositories-with-unlimited-collaborators-available-to-all-github-accounts-and-changes-to-github-paid-plans/" target="_blank" rel="noopener">Private repositories with unlimited collaborators available to all GitHub...</a></h3>

  <p>Private repositories with unlimited collaborators available to all GitHub accounts, and changes to GitHub paid plans</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>

<blockquote>
<p>Organizations and individuals on GitHub Free can now create unlimited private repositories with unlimited collaborators</p>
</blockquote>
